### DISCUSSION TITLE
Bluetooth connection failure

### DISCUSSION THREAD

**Discussion by fmorshed52** [iRobotEducation/create3_docs#354] [Questions] [answered]
I bought the latest (new) Create 3 iRobot last week.  Set it up (charged, update the firmware, etc.).  But the Bluetooth (blue) light does not come on.

Per instructions, I opened the front, and see that the "switch" is already turned to Bluetooth side.  I rebooted.  Still no blue light.  From my PC I do see that the "Bluetooth & devices" tab says "Bluetooth - Discoverable as "BOOK-somenumber".  But, when I click Add device, and pick  Bluetooth, it cannot find the device.

What is surprising is that the laptop says that it is discoverable, but it cannot connect to it.  What is not surprising, of course, is that the blue light on the iRobot does not come on, hence it cannot connect to it.

I have rummaged through all the iRobot setup instructions. I followed these pages to setup:  https://edu.irobot.com/create3-setup

Any ideas?

**Comment by shamlian** [comment, marked as answer]
I have seen it happen once or twice that the adapter board can get dislodged during shipping. Can you try pulling it out (through the rear of the robot, removing the payload tray) and then pushing it back in to fully seat the edge connector (like putting in an NES cartridge)? When it is fully seated, that debossed "DD" text on the top of the adapter should no longer be visible through the top of the robot (with the faceplate removed).

**Comment by shamlian** [comment]
If this doesn't work, what other LEDs are illuminated on the adapter board?

**Comment by fmorshed52** [comment]
I pulled the 1.25 inch wide assembly on right hand side that holds the Bluetooth (Bluetooth symbol, and USB symbol), and snapped it up into place as you said.  No change.  As far as which lights show, only the circle light shows.  Here are two images:
![IMG_0091 440](https://user-images.githubusercontent.com/122117850/224839878-3654e134-b68f-4a25-a49c-308a768eab8b.jpg)

![IMG_0090 442](https://user-images.githubusercontent.com/122117850/224840036-d7b9432f-d03e-4cef-9cb7-669211d8bea4.jpg)

**Comment by shamlian** [comment]
I can see from that picture that your [adapter board](https://iroboteducation.github.io/create3_docs/hw/adapter/) is not fully seated. Could you pull it out again, and make sure you can see the contacts here? 
![adapter-board-connector](https://user-images.githubusercontent.com/5374318/224845624-348afa05-f214-4fee-90e0-161bbdf8b6a1.jpg)

I wonder if a protective plug might have accidentally been left in the connector which is stopping it from seating. When it is fully seated, that debossed "DD" text on the top of the adapter should no longer be visible, like this:
![adapter-board-seated](https://user-images.githubusercontent.com/5374318/224845583-b085773f-9900-4f74-abdc-714f13eb46d9.jpg)

**Comment by shamlian** [comment]
Also -- a note, once you have gotten things working -- the robot uses Bluetooth Low Energy and not Bluetooth Classic, so there will be no need to pair it with your computer through the operating system.

**Comment by fmorshed52** [comment]
Aha, yes, I had not pushed the adaptor board all the way in.  I can see the blue and yellow light now.  Also, thanks for the note on the Bluetooth Low Energy.

Thank you!
